lib/persistence/abstract.js

```diff
@@ -138,6 +138,22 @@ AbstractPersistence.prototype.wire = function(server) {
   };
 };
 
+function arrayFrom(object) {
+  return [].slice.call(object);
+}
+
+/**
+ * Array.from polyfill
+ * @param aSet the set to changed to an array
+ */
+AbstractPersistence.prototype.arrayFrom = function(aSet) {
+  if (Array.from) {
+    return Array.from(aSet);
+  } else {
+    return arrayFrom(aSet);
+  }
+};
```

Comment:
there is no need for this to be in the prototype, it can be a separate function in a separate file.